src/observer.js

```diff
@@ -1,5 +1,11 @@
 import Viewport from './viewport'
 
+function ObserverCollection(opts) {
+  if (!(this instanceof ObserverCollection)) return new ObserverCollection(...arguments)
+  this.viewports = new Map()
+  this.handleScrollResize = opts.handleScrollResize || (h => h)
```

Comment:
I don't think you want the `|| (h => h)` because later you're checking if it was defined and it would always be true.

Reply:
```suggestion
  this.handleScrollResize = opts.handleScrollResize
```

Reply:
This comment still stands - you check it here: https://github.com/gpoitch/viewprt/pull/19/files#diff-7f04a433d8bd9a31fbfc7d21bedc46e1R57
We don't want the overhead of another function call when not using this feature.

Reply:
Got it!